sycl/tools/get_device_count_by_type.cpp

```diff
@@ -9,106 +9,204 @@
 #include <CL/cl.h>
 #include <CL/cl_ext.h>
 
+#include <cstdlib>
```

Comment:
Please, move this include to other standard C++ includes.

Reply:
Will do.